app/components/delete-reminder.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  store: Ember.inject.service(),
+
+  actions: {
+    delete(reminder){
+      debugger;
+      this.get("store").findRecord("reminder", this.reminder.id, {reload: true})
```

Comment:
Yes, we put in this {reload: true} because without it you receive this error: 
<img width="680" alt="screen shot 2017-02-16 at 9 52 13 am" src="https://cloud.githubusercontent.com/assets/15061527/23031276/a0913642-f42d-11e6-96c9-ea301de203ba.png">

So when I got that I researched it and found this page: https://github.com/samselikoff/ember-cli-mirage/issues/853

Basically, we're making a callback to the store and we need {reload: true} so that the code runs destroyRecord after the store responds.